spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/SpatialConnect.java

```diff
@@ -132,12 +165,37 @@ public void restartAllServices() {
         }
     }
 
+    public Observable<SCServiceStatusEvent> serviceStarted(final String serviceId) {
+        return Observable.create(new Observable.OnSubscribe<SCServiceStatusEvent>() {
```

Comment:
This Observable creation isn't necessary. You can return the event in `Observable.just(evt)`.